tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = py26, py27, py32, py33, py34
+envlist = py26, py27, py32, py33, py34, flake, flake_docs
 
 [testenv]
-commands = python tests.py
+commands = python tests.py
```

Comment:
Text files on Unix are supposed to have one trailing newline at the end of the last line in this file.  Please revert this bit also.